### ISSUE TITLE
Informing Players when they were offline when they got a warn

### ISSUE THREAD

**Issue by Morsmorse** [DevLeoko/AdvancedBan#227]
At the moment users don't notice when they get warned while being offline. A notification would be very useful when they join:

> You got a warning for xyz while you were offline. This is your nth warning. See /warns

Like that players learn from warnings more even if they are offline and the punisment doesn't remain unnoticed.

(sorry for the issue-spam, this is the last one, i swear ♥)

**Comment by TheKrafter** [comment]
yes, this would be very helpful. or even a papi placeholder that is either empty, or has a list of warns to put in something like the essentialsx `motd.txt`
